### ISSUE TITLE
A licence? A fully static class? 

### ISSUE THREAD

**Issue by ghost** [desean1625/php-Struct#1]
Please considered the following:

1- Can you add a licence?
2- I think to keep even more compatibility, your class need to be a fully statically class.

**Comment by desean1625** [comment]
License added. I will look into making it fully static,  I think it's a good idea but it will be a fair amount of work.

**Comment by ghost** [comment]
Yes thanks, but this is a very useful code it deserve your fully love, i think. It a shame that nobody find it here. In stack overflow there are lot of questions about migrate the format from python to php or from php to python. Do you think this questions will be there if they know about your code? Not way, it's really difficult understand both manuals about it and then find a way to convert one format to another. You really invert a lot of time here, your code line by line it's not so big, but what internally is involved it's really analize and underestand a lot.

**Comment by desean1625** [comment]
If I have time I'll look into it.

**Comment by ghost** [comment]
https://github.com/lestcape/DbfPHP
